README.md

```diff
@@ -6,11 +6,27 @@
 npm install
 
 npm run watch
+
+node server/index.js
 ```
+
 Then open the project in Visual Studio Code.
 Go to `extension.ts` then press F5.
 
-## Commands
+## To install as an extension
+
+```
+npm install -g @vscode/vsce
+
+cd live-interviewing
```

Comment:
I _think_ this doesn't need to be said, the assumption is that the user would already be in the root directory of the repo